keras_nlp/models/task.py

```diff
@@ -30,9 +29,9 @@ class Task(PipelineModel):
     """Base class for Task models."""
 
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
         self._backbone = None
         self._preprocessor = None
+        super().__init__(*args, **kwargs)
```

Comment:
curious - Why do we need to move the order here?

Reply:
Should I swap this for KerasCV as well? Not totally sure I understand the problem.